amacc.c

```diff
@@ -324,7 +324,7 @@ void next()
         case '\r':
             break;
         case '/':
-            if (*p == '/') { // comment
+	        if (*p == '/') { // comment
```

Comment:
Don't change this line.

Reply:
I found this is the issue for my vscode.